certbot/certbot/util.py

```diff
@@ -587,11 +598,11 @@ def is_ipaddress(address):
             return False
 
 
-def is_wildcard_domain(domain):
+def is_wildcard_domain(domain: str) -> bool:
```

Comment:
Should `domain` be `Union[str, bytes]`? The function body appears to accept either.

Reply:
Yes, fixed.